sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -176,9 +176,6 @@ export enum PiiEntityDomainType {
 // @public
 export type RecognizeCategorizedEntitiesErrorResult = TextAnalyticsErrorResult;
 
-// @public
-export type RecognizeCategorizedEntitiesOptions = TextAnalyticsOperationOptions;
```

Comment:
Removing the type would be breaking change. Someone might be using this type in their code and after updating to the latest, their code will fail to build

Reply:
ah right, I am not too bothered by it so will close for now.